src/queries/athena/changelog_buildings_join.sql

```diff
@@ -1,10 +1,13 @@
-SELECT release.id, release.names.primary AS primary_name, release.sources[1].dataset AS primary_source, change_type, ST_GeomFromBinary(geometry) AS geometry
-FROM v2024_06_13_beta_0 AS release
-        INNER JOIN changelog_v2024_06_13_beta_0 AS changelog ON release.id = changelog.id
+SELECT
+    release.id,
+    release.names."primary" AS primary_name,
+    release.sources [ 1 ].dataset AS primary_source,
+    change_type,
+    ST_GEOMFROMBINARY(geometry) AS geometry
+FROM v2024_08_20_0 AS release
+    INNER JOIN changelog_v2024_07_22_0 AS changelog ON release.id = changelog.id
```

Comment:
This is pointing to an out of date changelog and not dynamically pointing at the releases. I'll add a small commit to this PR to fix this.

Reply:
hold up. I pointed to an older release specifically for the example in that doc. I have some local edits to that doc to push tonight or tomorrow.